recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}   # required for C++ headers
+    - cmake
+    - make
+    - maven
+    - openjdk 8.*
+    - wget
+    - clangdev >=6.*
+    - go
+    - bisonpp
+    - flex
+  host:
+    - snappy  # required for mapd-core > 4.5.0
+    - arrow-cpp >=0.12
+    - blosc
+    - boost-cpp >=1.68.*
+    - double-conversion
+    - libgdal
+    - gflags
+    - glog
+    - libarchive
+    - libkml
+    - libpng
+    - llvmdev >=6.*
+    - openssl
+    - parquet-cpp
+    - thrift-cpp >=0.11.0
+  run:
+    - boost-cpp >=1.68.*
+    - libgdal
+    - glog
+    - gflags
+    - openjdk 8.*
+    - ncurses
+    - xz
+    - bzip2
+    - zlib
+about:
+  home: https://www.mapd.com/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.md
+  summary: 'The MapD Core database'
+
+  description: |
+    MapD Core is an in-memory, column store, SQL relational database 
+    that was designed from the ground up to run on GPUs.
+  doc_url: https://www.mapd.com/docs/
+  dev_url: https://github.com/mapd/mapd-core
+
+extra:
+  recipe-maintainers:
+    - xmnlab
+    - pearu
+    - andrewseidl
```

Comment:
@andrewseidl - are you OK with being a maintainer here?

Reply:
Yes, thanks!